edit.html

```diff
@@ -33,10 +34,15 @@
 
     <script src="vendor/codemirror/addon/edit/matchbrackets.js"></script>
 
-    <link rel="stylesheet" href="vendor/codemirror/addon/lint/lint.css" />
+    <!-- CSSLint -->
```

Comment:
Well, you can add a third option "none" to the linters selector. It would be internally represented as an empty string in `prefs` and `<option value="" i18n-text="genericNoneLabel">` in html. And genericNoneLabel in _locales/en/messages.json.